source/plugin/Assets/GoogleMobileAds/Platforms/Unity/DummyUtils

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2020 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections.Generic;
+using UnityEngine;
+using GoogleMobileAds.Api;
+
+namespace GoogleMobileAds.Unity
+{
+    public class DummyUtils
```

Comment:
all this can go in base class, eg AdBehavior

Reply:
if not moving base class, can we rename this to something more suitable like BaseAdDummyClient ?